common/achievement/util.ts

```diff
@@ -95,8 +112,11 @@ export async function getBucketContext(userID: string, relation?: string): Promi
 }
 
 export function transformPrismaJson(user: User, relation: string | null, json: Prisma.JsonObject): AchievementContextType {
-    // TODO: find proper type?
-    const transformedJson: any = { user: user };
+    const transformedJson: AchievementContextType = {
```

Comment:
The result of this function is not a `AchievementContextType`. It's a map with random key: value pairs.

Reply:
A type ExtendedAchievementContextType was added, taking any string as key and referencing any value it gets passed.